examples/vae/requirements.txt

```diff
@@ -1,4 +1,6 @@
 jax
+jaxlib
 flax
 tensorflow
 pillow
+git+https://github.com/google/ml_collections
```

Comment:
ml-collections is available as a pip package :) 

https://pypi.org/project/ml-collections/